### PR TITLE
add benchmark of rpc

### PR DESCRIPTION
**Pull request by mond77**
modify #54 
compare rpc performance with grpc in different RPC counts.

found result

create client and RPC 50000 
Screen Shot 2023-02-02 at 14 12 04

create client and RPC in 10、100、1000、10000
Screen Shot 2023-02-02 at 14 12 46

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -15,6 +15,7 @@ thiserror = "1.0.37"
 log = "0.4.17"
 clap = { version = "=4.0.18", features = ["derive"] }
 env_logger = "0.9.1"
+tonic = "0.8.2"
```
Comment (uran0sH) [inline Cargo.toml:18]: use dev-dependency

File: benches/grpc/grpc_server.rs [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+#[derive(Debug, Default)]
+pub struct MyGreeter {}
+
+#[tonic::async_trait]
+impl Greeter for MyGreeter {
+    async fn say_hello(
+        &self,
+        request: Request<HelloRequest>, // 接收以HelloRequest为类型的请求
+    ) -> Result<Response<HelloReply>, Status> {
+        // 返回以HelloReply为类型的示例作为响应
```
Comment (uran0sH) [inline benches/grpc/grpc_server.rs:20]: comment with English

File: benches/grpc/main.rs [hunk 3]
```diff
@@ -0,0 +1,26 @@
+mod grpc_client;
+mod grpc_server;
+
+use criterion::{criterion_group, criterion_main, Criterion};
+use grpc_client::gcli;
+use grpc_server::server;
+
+fn grpc_benchmark(c: &mut Criterion) {
+    std::thread::spawn(|| {
```
Comment (uran0sH) [inline benches/grpc/main.rs:9]: Should the server and client run in different cores?

Comment (mond77) [reply benches/grpc/main.rs:9]: nice suggestion

**Comment by luanshaotong** [comment]
Is it necessary to create benches directory?

**Review by luanshaotong — APPROVED** [review]
LGTM

**Pull request merged**